projects/ccf-rui/src/app/shared/components/details-label/details-label.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="label-container">
```

Comment:
No need for this wrapper div. Styling can be done on the details-label element instead. See comment on styles!